handler/payloadAnalytics.js

```diff
@@ -14,9 +14,7 @@ export async function accept(chat) {
 
     const thanksAnalytics = await getFaq(chat, 'thanksAnalytics');
 
-    if (chat.trackingEnabled) {
-        await chat.track.event('Analytics', 'Allowed', chat.language).send();
-    }
+    await chat.track.event('Analytics', 'Allowed', chat.language).send();
```

Comment:
Please revert this change. The performance benefit is practically non-existent, and it's better to be consistent in checking this property in case we want to disable tracking for reasons other than the user disagreeing. btw the :ok_hand: gitmoji is for changes due to code reviews

Reply:
`chat.trackingEnabled` is a boolean property and is loaded by default for every request. Checking it does not cause a DB request or anything

Reply:
`chat.trackingEnabled` is `false` in this case, which is why this check needs to be removed